crates/semantics/src/walker.rs

```diff
@@ -1038,9 +1038,9 @@ impl SemanticsWalker {
                         last_visibility = Some(modifier.span());
```

Comment:
```suggestion
                        last_write_visibility = Some(modifier.span());
```

this looks like a bug ( by me ) 😅

Reply:
no this looks fine, there's no `last_write_visibility` var in that method 👍

Reply:
Ah, my bad :)